API/Controllers/AuthController.cs

```diff
@@ -108,6 +108,25 @@ public async Task<ActionResult<AuthResponseDto>> ChangePassword([FromBody] Chang
         return this.Ok(result);
     }
 
+    /// <summary>
+    /// Changes the nickname of the currently authenticated user.
+    /// </summary>
+    /// <param name="request">The change-nickname request containing current and new nickname.</param>
+    /// <returns>The authenticated user's data and token.</returns>
+    /// <response code="200">The user nickname has been changed successfully.</response>
+    /// <response code="400">The request payload is invalid.</response>
+    /// <responce code="404">The authenticated user was not found.</responce>
```

Comment:
```suggestion
    /// <response code="404">The authenticated user was not found.</response>
```

Można też poprawić tę literówkę w innych metodach w tym pliku.